Modules/_ssl.c

```diff
@@ -1050,6 +1050,107 @@ _ssl__SSLSocket_do_handshake_impl(PySSLSocket *self)
     return NULL;
 }
 
+#if defined(TLS1_3_VERSION) && !defined(OPENSSL_NO_TLS1_3)
+/*[clinic input]
+_ssl._SSLSocket.key_update
+    updatetype: int
+    /
+
+[clinic start generated code]*/
+
+static PyObject *
+_ssl__SSLSocket_key_update_impl(PySSLSocket *self, int updatetype)
+/*[clinic end generated code: output=2eb3fb43e045bb4f input=338af2b9ff3d49eb]*/
+{
+    int ret;
+
+    PySSL_BEGIN_ALLOW_THREADS
+    ret = SSL_key_update(self->ssl, updatetype);
+    _PySSL_UPDATE_ERRNO_IF(ret == 0, self, ret);
+    PySSL_END_ALLOW_THREADS
+
+    if (PyErr_CheckSignals())
```

Comment:
Use `{` even for single statement `if`s (as per updated PEP 7)

Reply:
OK! Thanks for the comments.